File: Arcade/Intro/C++/09_allLongestStrings.cpp
std::vector<std::string> allLongestStrings(std::vector<std::string> inputArray) {
    int longestString=0;
    vector<string> result;
        for(string i : inputArray){
                if(i.length()>longestString){
                        result.clear();
                        longestString=i.length();
                        result.emplace_back(i);
                } else if(i.length()==longestString){
                        result.emplace_back(i);
                
                }
        }
        return result;
}

File: Arcade/Intro/JavaScript/09_allLongestStrings.js
function allLongestStrings(inputArray) {
    var longest = [""];
    
    for (var i = 0; i < inputArray.length; i++) {
        if (inputArray[i].length > longest[0].length) {
            longest = [inputArray[i]];
        } else if (inputArray[i].length === longest[0].length) {
            longest.push(inputArray[i]);
        }
    }
    
    return longest;
}
